src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+

```

Comment:
please remove redundant empty line

Reply:
i must remove line between package and public? If yes i can't because mvn clean package say this is error